src/seedu/addressbook/data/AddressBook.java

```diff
@@ -114,6 +123,12 @@ public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    public void setReminderMessage(String reminderMessage) {
+        this.reminderMessage = reminderMessage;
```

Comment:
Avoid the use of `this` outside of constructor, if need be, use a different name for your parameter